components/raftstore/src/router.rs

```diff
@@ -48,16 +49,26 @@ where
 
     /// Sends RaftCmdRequest to local store.
     fn send_command(&self, req: RaftCmdRequest, cb: Callback<EK::Snapshot>) -> RaftStoreResult<()> {
-        send_command_impl::<EK, _>(self, req, cb, None)
+        send_command_impl::<EK, _>(self, req, cb, None, DiskFullOpt::default())
     }
 
     fn send_command_with_deadline(
         &self,
         req: RaftCmdRequest,
         cb: Callback<EK::Snapshot>,
         deadline: Deadline,
+        disk_full_opt: DiskFullOpt,
```

Comment:
How about add a new `Options` to carry `deadline` and `disk_full_opt`?

Reply:
Done!